CITATION.cff

```diff
@@ -0,0 +1,123 @@
+cff-version: 1.2.0
+message: "If you use matRad, consider citing it and the corresponding research paper"
+authors:
+- family-names: "Ackermann"
+  given-names: "Benjamin"
+- family-names: "Bangert"
+  given-names: "Mark"
+- family-names: "Bennan"
+  given-names: "Amit Ben Antony"
+- family-names: "Burigo"
+  given-names: "Lucas"
+- family-names: "Cabal"
+  given-names: "Gonzalo"
+- family-names: "Cisternas"
+  given-names: "Eduardo"
+- family-names: "Charton"
+  given-names: "Louis"
+- family-names: "Christiansen"
+  given-names: "Eric"
+- family-names: "Ecker"
+  given-names: "Swantje"
+- family-names: "Ellerbrock"
+  given-names: "Malte"
+- family-names: "Gabryś"
+  given-names: "Hubert"
+- family-names: "Handrack"
+  given-names: "Josefine"
+- family-names: "Heath"
+  given-names: "Emily"
+- family-names: "Hermann"
+  given-names: "Cindy"
+- family-names: "Homolka"
+  given-names: "Noa"
+- family-names: "Jäkel"
+  given-names: "Oliver"
+- family-names: "Klinge"
+  given-names: "Thomas"
+- family-names: "Mairani"
+  given-names: "Andrea"
+- family-names: "Mescher"
+  given-names: "Hennig"
+- family-names: "Müller"
```

Comment:
Clean up

Reply:
Does this refer to the encoding issue? (which is not visisble here)?